hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenCancelAirdropHandlerTest.java

```diff
@@ -243,6 +246,86 @@ private PendingAirdropId[] repeatedAirdrops(final PendingAirdropId airDrop, fina
         }
     }
 
+    @Test
+    void pureChecksThrowsForDuplicateAirdropIds() {
```

Comment:
This test doesn't seem related to simple fees. Is it some other functionality that is not currently covered by tests?